src/qml/InformationDrawer.qml

```diff
@@ -65,10 +65,10 @@ Item {
       PositioningInformationView {
         id: positioningInformationView
         width: parent.width
-        height: contentHeight
+        height: Math.min(contentHeight, mainWindow.height / 3)
         visible: positioningInformationViewEnabled
         positionSource: controller.positionSource
-        antennaHeight: positioningSettings.antennaHeightActivated ? positioningSettings.antennaHeight : NaN
+        antennaHeight: positioningSettings.antennaHeightActivated ? positioningSettings.antennaHeight : -1
```

Comment:
-1 is valid. Negative antenna  height is allowed.

Reply:
Ok, so previously it was setting `NaN` and for altitude :

```QML
if (!isNaN(parseFloat(antennaHeight))) {
  details.push('ant.');
}
``` 

any suggestion ?

Reply:
i tried to assign `NaN` to a Q_PROPERTY with type `double`, it simply ignored it and value did not changed.